### PR TITLE
Eth <> Wei + ERC20 Formatting functions for Utils.cs

### PR DESCRIPTION
**Pull request by 0xFirekeeper**
ToWei, ToEth and FormatERC20 functions added.

### CODE REVIEW

**Comment by 0xFirekeeper** [comment]
Ok this should be good now

File: Assets/Thirdweb/Scripts/Utils.cs [hunk 1]
```diff
@@ -53,5 +55,37 @@ public static long UnixTimeNowMs()
...
+                return "INVALID_ARGUMENTS";
+            double eth = (double)weiBigInt / DECIMALS_18;
+            return eth.ToString();
+        }
+
+        public static string FormatERC20(this string wei, int decimalsToDisplay = 4, int decimals = 18)
+        {
+            BigInteger weiBigInt = 0;
+            if (!BigInteger.TryParse(wei, out weiBigInt))
+                return "INVALID_ARGUMENTS";
```
Comment (joaquim-verges) [inline Assets/Thirdweb/Scripts/Utils.cs:81]: probably want to throw here instead no?

File: Assets/Thirdweb/Scripts/Utils.cs [hunk 2]
```diff
@@ -53,5 +55,37 @@ public static long UnixTimeNowMs()
...
+            return wei.ToString();
+        }
+
+        public static string ToEth(this string wei)
+        {
+            BigInteger weiBigInt = 0;
+            if (!BigInteger.TryParse(wei, out weiBigInt))
+                return "INVALID_ARGUMENTS";
+            double eth = (double)weiBigInt / DECIMALS_18;
+            return eth.ToString();
```
Comment (joaquim-verges) [inline Assets/Thirdweb/Scripts/Utils.cs:74]: how about just calling `FormatERC20` here? that way we can expose `decimalsToDisplay` as an optional param on `ToEth` too which is super useful

**Review by joaquim-verges — COMMENTED** [review]
looks good! just a couple of comments

File: Assets/Thirdweb/Scripts/Utils.cs (hunk 1 above)
Comment (0xFirekeeper) [reply Assets/Thirdweb/Scripts/Utils.cs:81]: Up to you, wasn't sure what's best

File: Assets/Thirdweb/Scripts/Utils.cs (hunk 2 above)
Comment (0xFirekeeper) [reply Assets/Thirdweb/Scripts/Utils.cs:74]: Yep good point!

**Comment by 0xFirekeeper** [comment]
Here we go!

**Review by joaquim-verges — APPROVED** [review]
amazing, thank you!

**Pull request merged**